charts/seed-monitoring/charts/grafana/values.yaml

```diff
@@ -19,4 +19,26 @@ extensions:
   dashboards: ""
 
 konnectivityTunnel:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+exposedComponents:
+- dashboardName: "Logging Kube Apiserver"
```

Comment:
Can we remove the logging prefix from the dashboards? I don't think the dashboard is only useful for logging. WDYT?

Reply:
I did this prefix to make it easier for users to find the dashboards that have logs

Reply:
You can also add a `logging` tag to the dashboard

Reply:
Done :)